src/image_bios.cpp

```diff
@@ -5,10 +5,9 @@
 
 #include "config.h"
 
-#include "image_bios.hpp"
-
 #include "dbus.hpp"
 #include "fwupderr.hpp"
+#include "image_bios.hpp"
```

Comment:
Leave it as it was before, please. :)
Probably we need to review .clang-fromat for the new clang version.